lib/actions/api.js

```diff
@@ -561,6 +585,15 @@ function createQueryAction (endpoint, responseAction, errorAction, options) {
       const api = otpState.config.api
       url = `${api.host}${api.port ? ':' + api.port : ''}${api.path}/${endpoint}`
     }
+
+    // don't make a request to a URL that has already seen the same request
+    // within the last 10 seconds
+    if (throttledUrls[url] && throttledUrls[url] > now() - TEN_SECONDS) {
+      // URL already had a request within last 10 seconds, exit
```

Comment:
Should this log a warning? My impression is yes. Otherwise, future developers will probably have no idea what is going on when they can't figure out why a request is short circuiting.

Reply:
Will add